blog/content/post/2020-11-30-coreboot_for_power.md

```diff
@@ -0,0 +1,154 @@
+---
+title: coreboot port for OpenPOWER - why bother?
+abstract: 'You may have heard by now that we are working on coreboot port for
+          Talos II. OpenPOWER already has, nomen omen, open source firmware, so
+          one may ask why bother? We will try to answer that question.'
+cover: /covers/coreboot-logo.svg
+author: krystian.hebel
+layout: post
+published: true
+date: 2020-11-30
+archives: "2020"
+
+tags:
+  - firmware
+  - coreboot
+  - openpower
+categories:
+  - Firmware
+
+---
+
+You may have heard by now that we are working on coreboot port for [Talos II](https://raptorcs.com/TALOSII/).
+OpenPOWER already has, nomen omen, open source firmware, so one may ask why
+bother? In this blog post we will try to answer that question.
```

Comment:
@KBanecka I would add couple references here OpenPOWER Summit, OSFC, maybe some tweet.

Reply:
nomen omen? never heard this phrase before.

Reply:
It should rather be `nomen atque omen` or `nomen est omen`. Nomen omen is an abbreviation used often in polish